src/components/Main.vue

```diff
@@ -189,6 +187,11 @@ export default {
                     this.messages = this.getMessages(text, flags, messages);
                     this.summaries = this.getSummaries(issues);
                 });
+
+            this.sidebarStatus = true;
+            this.$emit('hideHeader');
+            this.changeWidth();
+
```

Comment:
I see what you were trying to do here, but it's not the most elegant solution. The problem, which you'll learn about in CS 370, is that you now have code within `renderIssues` at multiple different levels of abstraction.

A better solution is to write a new method that calls the existing methods, like so:

<pre>
newMethod() {
  this.renderIssues();
  this.sidebarStatus = true;
  this.$emit('hideHeader');
  this.changeWidth();
}
</pre>

Do you see how the code for that new method is all at a similar level of abstraction? What should the new method be named?

Reply:
@ProfJanetDavis Yes, that makes a lot of sense. I suppose we could name it `render`? Or is that not descriptive enough?